microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/SubsystemDefinition.java

```diff
@@ -0,0 +1,30 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import java.util.Arrays;
+import java.util.Collection;
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.PersistentResourceDefinition;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+
+public class SubsystemDefinition extends PersistentResourceDefinition {
+    static AttributeDefinition[] ATTRIBUTES = {};
+
+    protected SubsystemDefinition() {
+        super(
+            SubsystemExtension.SUBSYSTEM_PATH,
+            SubsystemExtension.getResourceDescriptionResolver(SubsystemExtension.SUBSYSTEM_NAME),
+            SubsystemAdd.INSTANCE,
+            SubsystemRemove.INSTANCE
+        );
+    }
+
+    @Override
+    public Collection<AttributeDefinition> getAttributes() {
+        return Arrays.asList(ATTRIBUTES);
+    }
+
+    @Override
+    public void registerOperations(ManagementResourceRegistration resourceRegistration) {
+        super.registerOperations(resourceRegistration);
+    }
```

Comment:
This is a bit nitpicky, but there probably isn't really a need to override this.